src/main/java/edu/greenblitz/robotName/subsystems/arm/roller/neoRoller/NeoRoller.java

```diff
@@ -50,11 +45,18 @@ public void moveToPosition(Rotation2d position) {
         );
     }
 
+    public boolean isObjectInByCurrent(double current){
+        return current - lastInputs.outputCurrent >= NeoRollerConstants.NOTE_IN_CURRENT_JUMP
```

Comment:
We need to filter out the motor start current (a short spike in current) maybe a low cut filter, or debouncer or a flag for after the roller reach cruise velocity and only if the rollers are close to the setpoint and the current spikes then consider it object in